app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,33 @@
+module SessionsHelper
+  def log_in(user)
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+    @current_user = user
+  end
+
+
+  # Returns the user corresponding to the remember token cookie.
+  def current_user
+      @current_user = User.find_by(id: cookies.signed[:user_id])
+  end
```

Comment:
[OPTIONAL]

Consider memoizing `@current_user` so you don't need to hit the database if @current had already been set. 

=> `@current_user ||= User.find_by(id: cookies.signed[:user_id])`